packages/bottender/src/slack/__tests__/SlackEvent.spec.ts

```diff
@@ -551,6 +551,21 @@ const interactiveMessageEvent = {
   triggerId: '274927463524.5223114719.95a5b9f6d3b30dc7e07dec6bfa4610e5',
 };
 
+const slashCommandMessageEvent = {
```

Comment:
how about adding a fixture for slash command without arguments?

Reply:
@darkbtf

Reply:
ok, I'll add that test case